com.unity.postprocessing/PostProcessing/Runtime/PostProcessManager.cs

```diff
@@ -116,52 +113,60 @@ public void GetActiveVolumes(PostProcessLayer layer, List<PostProcessVolume> res
             var triggerPos = onlyGlobal ? Vector3.zero : volumeTrigger.position;
 
             // Sort the cached volume list(s) for the given layer mask if needed and return it
-            var volumes = GrabVolumes(mask);
-
-            // Traverse all volumes
-            foreach (var volume in volumes)
+            using (ListPool<PostProcessVolume>.Get(out var volumes))
             {
-                // Skip disabled volumes and volumes without any data or weight
-                if ((skipDisabled && !volume.enabled) || volume.profileRef == null || (skipZeroWeight && volume.weight <= 0f))
-                    continue;
-
-                // Global volume always have influence
-                if (volume.isGlobal)
+                Exception e;
+                if ((e = m_Volumes.FindSortedVolumesByLayerMask(mask, volumes)) != null)
                 {
-                    results.Add(volume);
-                    continue;
+                    Debug.LogException(e);
```

Comment:
Should we have this in the final PR?

Reply:
I prefer to log an error if any occurs instead of silently fail. So I would keep it in the final PR